plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/validation/XtendJavaValidator.java

```diff
@@ -932,6 +932,7 @@ private boolean contributesToConflict(JvmDeclaredType type, Set<JvmDeclaredType>
 	protected void reportMissingImplementations(XtendTypeDeclaration xtendClass, JvmGenericType inferredType, List<IResolvedOperation> operationsMissingImplementation) {
 		StringBuilder errorMsg = new StringBuilder();
 		String name = xtendClass.getName();
+		boolean needsNewLine = operationsMissingImplementation.size() > 1;
 		if (xtendClass.isAnonymous()) {
 			JvmTypeReference superType = Iterables.getLast(inferredType.getSuperTypes());
 			errorMsg.append("The anonymous subclass of ").append(superType.getSimpleName());
```

Comment:
Right! I fixed this now, but I'm not sure how to best submit it. 

If I add a commit to the original branch, it should be possible to merge again, but I don't know if GitHub will detect that the PR changed status. Alternatively, I can open a new PR. 

What do you think? Shall we try updating this one and see if it works?